reana_server/utils.py

```diff
@@ -99,6 +99,7 @@ def _create_user(email, user_access_token, admin_access_token):
         user_parameters = dict(access_token=user_access_token)
         user_parameters["email"] = email
         user = User(**user_parameters)
+        user.initialize_user_quota_limits()
```

Comment:
Is this operation going to be needed **every time** we create a user? If that is the case, would it be possible to move it inside the `User` model constructor?

What about the users created via UI? Are we supposed to do this initialization manually? (Sorry if this was already discussed, I probably missed it)

Reply:
Thanks for spotting these things!

I agree 💯 %  with you. This is one of the few things I am not super happy about... The problem is that this `initialize_user_quota_limits` needs the User to be committed first... that's why initially I just created the method and went this way, so I would wrap up. I will have a look and see if I can figure it out.

Regarding the UI I overlooked it thinking that `_create_user` would be used there but actually [it isn't](https://github.com/reanahub/reana-server/blob/c4eb0f9a317a9701538e28b4abc25e0d7f6abb9b/reana_server/utils.py#L167), I will fix it.

Reply:
Yeah.. too bad that we're not using the same function for CLI/API and UI creation of users... Maybe we can unify in the future?